sql/parser/sql.go

```diff
@@ -333,313 +333,319 @@ var yyExca = []int{
 	-2, 0,
```

Comment:
Yep. That worked. I need to make the `go generate` command add that automatically. Should be easy.